src/actions.sol

```diff
@@ -94,6 +98,11 @@ contract Actions is DSNote {
         shelf.repay(loan, amount);
```

Comment:
should we switch the approve and transferFrom in the repay function.

because we approve currency we don't have yet. maybe it is not a problem

Reply:
we can